### PR TITLE
convert any paths into absolute which file monitor expects

### PR DESCRIPTION
**Pull request by pushad**
On windows I've discovered that frida-tools (CLI) does not auto reload when using relative paths.
Making use of the os.path.abspath we convert the relative path into an absolute path.

### CODE REVIEW

File: frida_tools/repl.py [hunk 1]
```diff
@@ -172,7 +173,7 @@ def _monitor_script(self):
             self._unmonitor_script()
 
             if self._user_script is not None:
-                monitor = frida.FileMonitor(self._user_script)
+                monitor = frida.FileMonitor(abspath(self._user_script))
```
Comment (oleavr) [inline frida_tools/repl.py]: Would be good to do this in `_initialize` instead:

```Python
        def _initialize(self, parser, options, args):
            self._user_script = os.path.abspath(options.user_script)
```

(And we can use the `os` import that we already have, as the function is only used once and not in a performance-critical path.)

Comment (pushad) [reply frida_tools/repl.py]: That makes sense. Thanks for the update.
Will modify and update when I can!

Comment (pushad) [reply frida_tools/repl.py]: Applied suggested changes.

File: frida_tools/repl.py [hunk 2]
```diff
@@ -67,7 +66,7 @@ def _add_options(self, parser):
             parser.add_option("-o", "--output", help="output to log file", dest="logfile", default=None)
 
         def _initialize(self, parser, options, args):
-            self._user_script = options.user_script
+            self._user_script = os.path.abspath(options.user_script)
```
Comment (oleavr) [inline frida_tools/repl.py]: I guess this will crash if `options.user_script` is `None` because `-l` wasn't specified?

**Comment by oleavr** [comment]
Fixed in 3ca7f468a7158f3f307132a55b31a14e5889a6ee.

**Pull request closed without merging**